src/components/Navbar/Navbar.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { Navbar } from './Navbar';
+
+const content = 'navbar content';
+
+storiesOf('Navbar', module)
+	.addParameters({ jest: ['Container'] })
+	.add('Navbar', () => (
+		<Fragment>
+			<Navbar>{content}</Navbar>
+			<div style={{ height: '200px' }} />
+			<Navbar placement="bottom">{content}</Navbar>
+		</Fragment>
+	))
+	.add('Fixed navbar', () => (
+		<Fragment>
+			<Navbar position="fixed" background={'alt'}>
+				{content}
+			</Navbar>
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+		</Fragment>
+	))
+	.add('Auto height navbar', () => (
+		<Navbar autoHeight={true}>
+			<Fragment>{content}</Fragment>
+			<br />
+			<Fragment>{content}</Fragment>
+			<br />
+			<Fragment>{content}</Fragment>
+			<br />
+			<Fragment>{content}</Fragment>
+			<br />
+			<Fragment>{content}</Fragment>
+		</Navbar>
+	))
+	.add('Navbar backgrounds', () => (
+		<Fragment>
+			<Navbar>{`${content} on a transparent background`}</Navbar>
+			<br />
+			<Navbar background="white">{`${content} on a 'white' background`}</Navbar>
+			<br />
+			<Navbar background="alt">{`${content} on an 'alt' background`}</Navbar>
+			<br />
+			<Navbar background="inverse">{`${content} on an 'inverse' background`}</Navbar>
```

Comment:
mss de content nog wrappen en witte color geven (zoals bij inverse container)
`<p style={{ color: 'white' }}>{content}</p>`

Reply:
ik dacht ook van dat de doen, maar dan zou iemand die de storybook bekijkt mss denken dat de color automatisch op wit gezet wordt. Nu is het duidelijk dat dit de verantwoordelijkheid is van de user die de component  gebruikt.

Reply:
Je kunt op zicht wel zien in de jsx-tab van de story. Maar okay, kan ik mij in vinden.